src/core.h

```diff
@@ -289,6 +289,7 @@ enum {STREAM_PRESZ = 4+12}; /* same as RTP_HEADER_SIZE */
 typedef void (stream_rtp_h)(const struct rtp_header *hdr,
 			    struct rtpext *extv, size_t extc,
 			    struct mbuf *mb, unsigned lostc, void *arg);
+typedef int (stream_pt_h)(uint8_t pt, struct mbuf *mb, void *arg);
```

Comment:
is it possible to make a separate PR for this change, including the code that is
handling payload-type changes?

Reply:
I'll try so. Anyway, I forgot to call s->pth() during preparing the PR.

Reply:
Currently try to divide the big commit into:
- pt-handler
- silence detection
- stream decode